Makefile.armv7a

```diff
@@ -10,12 +10,16 @@ CROSS ?= arm-phoenix-
 
 CC := $(CROSS)gcc
 
+DEFAULT_OLVL := -O2
 ifeq ($(DEBUG), 1)
-  CFLAGS += -Og
+  OLVL := -Og
```

Comment:
we don't have precedence of naming the build params You can provide externally by env variables, maybe we should make some rule in the future (adding comment just for discussion)

Reply:
Agree